usage.txt

```diff
@@ -26,6 +26,7 @@ out                the dir to put the app into at the end. defaults to current w
 overwrite          if output directory for a platform already exists, replaces it rather than skipping it
 prune              runs `npm prune --production` on the app
 sign               should contain the identity to be used when running `codesign` (only for building for the darwin platform, on OS X)
+strict-ssl         whether electron should be downloaded with strict-ssl (defaults to true)
```

Comment:
Capitalize "Electron", and this should probably be reworded to not say `strict-ssl` in the description.